### ISSUE TITLE
Feature Request: Support ZodBranded

### ISSUE THREAD

**Issue by erikhofer** [matejchalk/zod2md#5]
Example:
```
import { z } from 'zod'

export const Foo = z
  .string()
  .brand<'Foo'>()
```
Leads to the following error:
```
Error: Zod type ZodBranded is not supported
    at convertSchema (file://xxx/node_modules/zod2md/dist/bin.js:170:9)
    at file://xxx/node_modules/zod2md/dist/bin.js:39:8
    at Array.map (<anonymous>)
    at convertSchemas (file://xxx/node_modules/zod2md/dist/bin.js:36:26)
    at zod2md (file://xxx/node_modules/zod2md/dist/bin.js:1075:18)
    at async runCLI (file://xxx/node_modules/zod2md/dist/bin.js:1155:20)
```

Is it possible to add support for this?

**Comment by matejchalk** [comment]
> Is it possible to add support for this?

Yes :+1: I didn't get round to adding support for some of the more obscure Zod types yet, but hope to find some time for it soonish.

**Comment by matejchalk** [comment]
Released in version [0.1.3](https://github.com/matejchalk/zod2md/releases/tag/0.1.3) :rocket:
